demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
```

Comment:
>  It was always working and everybody was doing it too without problems.

It is more that you didn't have use-cases that would expose problems. This doesn't mean that those don't exist.